MobileCenterDistribute/MobileCenterDistribute/Internals/Model/MSReleaseDetails.m

```diff
@@ -152,4 +153,35 @@ - (NSDictionary *)serializeToDictionary {
 - (BOOL)isValid {
   return (self.id && self.downloadUrl);
 }
+
+- (BOOL)isEqual:(id)object {
+  if (![object isKindOfClass:[MSReleaseDetails class]]) {
+    return NO;
+  }
+  MSReleaseDetails *details = (MSReleaseDetails *)object;
+  return ((!self.id && !details.id) || [self.id isEqualToNumber:details.id]) &&
+         ((!self.status && !details.status) || [self.status isEqualToString:details.status]) &&
+         ((!self.appName && !details.appName) || [self.appName isEqualToString:details.appName]) &&
+         ((!self.version && !details.version) || [self.version isEqualToString:details.version]) &&
+         ((!self.shortVersion && !details.shortVersion) || [self.shortVersion isEqualToString:details.shortVersion]) &&
+         ((!self.releaseNotes && !details.releaseNotes) || [self.releaseNotes isEqualToString:details.releaseNotes]) &&
+         ((!self.provisioningProfileName && !details.provisioningProfileName) ||
+          [self.provisioningProfileName isEqualToString:details.provisioningProfileName]) &&
+         ((!self.size && !details.size) || [self.size isEqualToNumber:details.size]) &&
+         ((!self.minOs && !details.minOs) || [self.minOs isEqualToString:details.minOs]) &&
+         (self.mandatoryUpdate == details.mandatoryUpdate) &&
+         ((!self.fingerprint && !details.fingerprint) || [self.fingerprint isEqualToString:details.fingerprint]) &&
+         ((!self.uploadedAt && !details.uploadedAt) || [self.uploadedAt isEqual:details.uploadedAt]) &&
+
+         // Don't compare downloadUrl. downloadUrl contains pltoken param which will have different values every time.
+         // ((!self.downloadUrl && !details.downloadUrl) || [self.downloadUrl isEqual:details.downloadUrl]) &&
+         ((!self.appIconUrl && !details.appIconUrl) || [self.appIconUrl isEqual:details.appIconUrl]) &&
```

Comment:
The reason of the change is that download URL only contains pltoken param which will be changed every request. other URLs won't be changed like download URL since those don't contain something like request token.